pkgs/development/python-modules/pdfposter/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, pypdf2 }:
+
+buildPythonPackage rec {
+  pname = "pdftools.pdfposter";
+  version = "0.7.post1";
+
+  propagatedBuildInputs = [ pypdf2 ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0c1avpbr9q53yzq5ar2x485rmp9d0l3z27aham32bg7gplzd7w0j";
+  };
+
+  meta = with stdenv.lib; {
+    description = "pdfposter is much like poster does for Postscript files, but working with PDF.";
```

Comment:
`meta.description` should:

*    Be capitalized.
*    Not start with the package name.
*    Not have a period at the end.
(see https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md)

Reply:
Ha, managed to violate all three points with a single commit. Fixed now.